tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/OpenTelemetrySdkTests.cs

```diff
@@ -213,10 +215,11 @@ public async Task IntegrationDisabled(string packageVersion)
         [SkippableTheory]
         [Trait("Category", "EndToEnd")]
         [Trait("RunOnWindows", "True")]
+        [Trait("RequiresDockerDependency", "true")]
```

Comment:
The tests also failed on Debian/Alpine so perhaps we don't have an arm64 image of the dd-apm-test-agent? If that's the case, we should indicate that we can't run this on arm

```suggestion
        [Trait("RequiresDockerDependency", "true")]
        [Trait("Category", "ArmUnsupported")]
```